packages/core/mesh/messaging/src/messenger.test.ts

```diff
@@ -313,4 +317,39 @@ describe('Messenger', () => {
       expect(count).toEqual(1);
     });
   });
+
+  describe.skip('load', () => {
```

Comment:
skip?

Reply:
just wanted to keep the code here in case we need load-test signal

Reply:
not really a test that should be running